src/oralb_ble/parser.py

```diff
@@ -287,3 +293,32 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         self.update_binary_sensor(
             str(OralBBinarySensor.BRUSHING), bool(state == 3), None, "Brushing"
         )
+
+    async def async_poll(self, ble_device: BLEDevice) -> SensorUpdate:
```

Comment:
Can you add a test for this that locks out the bleak and bleak retry connector calls?

Reply:
When you say lock, are you talking about pytest-locker or pytest mocking?

Reply:
Should have been mocks.  Autocorrect 🤣